woocommerce/woocommerce
# Added storing attribute values as post_excerpt for variations.

This allows searching for variations by their attributes without cluttering the Variation's title.
If the target version will be different than 3.5.3, the db routine needs to be updated.

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/wc-admin/issues/831 .

### How to test the changes in this Pull Request:

1. Create a variable product with 4 attributes.
2. Create variations for the product.
3. Try to search for the variations of the parent product by attribute value, e.g. color via REST API: `https://local.wordpress.test/wp-json/wc/v3/products/1168/variations/?search=black`
4. Old version won't work, new version should work fine.

### Other information:




### Changelog entry

> Added storing attribute values as post_excerpt for variations to support easier searching for variations.
